app/src/main/java/com/celzero/bravedns/service/BraveVPNService.kt

```diff
@@ -937,6 +1008,18 @@ class BraveVPNService : VpnService(), ConnectionMonitor.NetworkListener, Protect
                 val builder = updateNotificationBuilder(this)
                 notificationManager.notify(SERVICE_ID, builder.build())
             }
+            PersistentState.APP_STATE -> {
```

Comment:
This shouldn't be a persistent state. It should be a local state in `BraveVPNService`.

Reply:
refactored.